core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -11,12 +11,7 @@
 {% block header_js %}
   {{ super() }}
   <script type="text/javascript">
-    GLOBALS.canEdit = JSON.parse('{{can_edit|js_string}}');
-    GLOBALS.canUnpublish = JSON.parse('{{can_unpublish|js_string}}');
     GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
-    GLOBALS.collectionTitle = JSON.parse('{{title|js_string}}');
```

Comment:
Removed (and this should address the first comment as well regarding passing the title to the front)